meta/lvar.rb

```diff
@@ -1,16 +1,10 @@
 # frozen_string_literal: true
 
 Mutant::Meta::Example.add :lvar do
-  source 'a = nil; a'
+  declare_lvar :a
```

Comment:
Not really an unparser artifact. But correctly using the parser API to avoid redundancy.